src/components/contactBottomHeaderArea.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from "react"
+import "./contactBottomHeaderArea.scss"
+
+
+class ContactBottomHeaderArea extends Component {
+    constructor(props) {
+        super(props);
+    
+        this.state = {
+          firstName: "",
+          lastName: "",
+          email: "",
+          subject: "",
+          message: "",
+        };
+      }
+
+    encode = (data) => {
+        return Object.keys(data)
+            .map(key => encodeURIComponent(key) + "=" + encodeURIComponent(data[key]))
+            .join("&")
+      }
+    
+    handleSubmit = (event) => {
+      event.preventDefault()
+      fetch("/", {
+        method: "POST",
+        headers: { "Content-Type": "application/x-www-form-urlencoded" },
+        body: this.encode({
+          "form-name": "contact-form",
+          ...this.state
+        })
+      }).then(() => alert("Success!"))
+        .catch(error => alert(error));
+
+      event.preventDefault();
+    }
+
+    handleChange = (event) => {
+        const { name, value } = event.target;
+        this.setState({ [name]: value });
+    }
+
+    render() {
+        const { 
+            firstName, 
+            lastName, 
+            email,
+            subject, 
+            message 
+        } = this.state;
+
+        const bottomLeft = (
+            <form 
+                name="contact-form" 
+                method="POST" 
+                data-netlify="true" 
+                netlify-honeypot="bot-field"
+                onSubmit={this.handleSubmit}
+            >
+                <input type="hidden" name="form-name" value="contact-form" />
+                <input type="hidden" name="bot-field" />
+                
+                <div class="form-row">
+                    <input type="text" name="firstName" placeholder="First Name" value={firstName} onChange={this.handleChange}/>
+                    <input type="text" name="lastName" placeholder="Last Name" value={lastName} onChange={this.handleChange}/>
+                </div>
+                <div class="form-row"> 
+                    <input type="text" name="email" placeholder="Email" value={email} onChange={this.handleChange}/>
```

Comment:
I thought you already updated this, but I want to make sure this is `type="email"` rather than text.